apps/framework-cli/src/cli.rs

```diff
@@ -561,7 +565,11 @@ async fn top_command_handler(
                 &settings
             );
 
-            list_all(project_arc, version, limit).await
+            if *streaming {
+                list_streaming(project_arc, limit).await
+            } else {
+                list_db(project_arc, version, limit).await
+            }
```

Comment:
What if the user wants to list "everything"?

Reply:
We could make --streaming flag additive instead of either-or, but would need to figure out what to show if they also add --version flag because right now we show all streaming topic versions per data model. Basically, need to iterate with product to see how this fits in their flow and if there are other ways we want to organize this information.